rally_ovs/plugins/ovs/ovsclients_impl.py

```diff
@@ -242,13 +244,36 @@ def flush(self):
 
         def add_port(self, bridge, port, may_exist=True):
             opts = ['--may-exist'] if may_exist else None
-            self.run('add-port', opts, [bridge, port])
+            self.run(" ovs-vsctl ", 'add-port', opts, [bridge, port])
 
 
         def db_set(self, table, record, *col_values):
             args = [table, record]
             args += set_colval_args(*col_values)
-            self.run("set", args=args)
+            self.run(" ovs-vsctl ", "set", args=args)
+
+        def of_check(self, bridge, port, mac_addr, may_exist=True):
+            in_port = ""
+            stdout = StringIO.StringIO()
+            self.run(" ovs-ofctl ", "show br-int", stdout=stdout)
+            show_br_int_output = stdout.getvalue()
+            in_port = get_of_in_port(show_br_int_output, port)
+            LOG.info("Check port: in_port: %s; mac: %s" % (in_port.strip(), mac_addr))
+            appctl_cmd = " ofproto/trace br-int in_port=" + in_port.strip()
```

Comment:
@hzhou8 thanks for reviewing the PR. You are right; binding operation could be verified by ```ofctl dump-flow```. However, IMO, tracing a flow, which is used [OVN tutorial](https://github.com/openvswitch/ovs/blob/master/tutorial/ovn/env3/packet1.sh#L19), seems more accurate and straightforward.